services/API-service/src/scripts/seed-init.ts

```diff
@@ -106,7 +106,7 @@ export class SeedInit implements InterfaceScript {
           countryEntity.adminRegionLabels = JSON.parse(
             JSON.stringify(country.adminRegionLabels),
           );
-          countryEntity.eapLink = country.eapLink;
+          countryEntity.eapLinks = JSON.parse(JSON.stringify(country.eapLinks));
```

Comment:
```suggestion
          countryEntity.eapLinks = JSON.parse(JSON.stringify(country.eapLinks));
          countryEntity.eapLinks = {...country.eapLinks} 
```

Reply:
@SwatiTh this suggestion gives me immediate red errors in VS code 

Type '{ floods: string; malaria?: undefined; "heavy-rain"?: undefined; dengue?: undefined; drought?: undefined; } | { floods: string; malaria?: undefined; "heavy-rain"?: undefined; dengue?: undefined; drought?: undefined; } | ... 4 more ... | { ...; }' is not assignable to type 'JSON'.
  Type '{ floods: string; malaria?: undefined; "heavy-rain"?: undefined; dengue?: undefined; drought?: undefined; }' is missing the following properties from type 'JSON': parse, stringify,